tests/queries/0_stateless/03010_read_system_parts_table_test.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS users;
```

Comment:
Could you please also add tests for `parts_columns`, `projection_parts` and `projection_parts_columns`? They all use the same base class, in theory they should behave the same, but better to have a test for them.

Reply:
Hi János, thanks for the suggestion! Just had a try, but found out that [`projection_parts`](https://github.com/ClickHouse/ClickHouse/blob/e82cf5d5ba62ec4e13288052e690228707e9de4e/src/Storages/System/StorageSystemProjectionParts.cpp#L21) and [`projection_parts_columns`](https://github.com/ClickHouse/ClickHouse/blob/e82cf5d5ba62ec4e13288052e690228707e9de4e/src/Storages/System/StorageSystemProjectionPartsColumns.cpp#L21) do not have a column called `uuid`. 

Added test for `projection_parts`.